src/main/java/org/usfirst/frc/team4373/robot/input/hid/RooJoystick.java

```diff
@@ -70,8 +95,8 @@ public double rooGetThrottle() {
     /**
      * Returns the filtered value of a joystick axis.
      *
-     * @param axis the axis from which to read.
-     * @return the filtered value of the axis.
+     * @param axis The axis from which to read.
```

Comment:
The first letter of JavaDoc descriptions *should* be lower case.

Reply:
Huh. I didn't know that.

Reply:
Well, technically, it's not *that* clear-cut (WPILib actually does capitalize). But the Java standard library doesn't, and we haven't been, so…convention.

Reply:
According to [Oracle](http://www.oracle.com/technetwork/articles/java/index-137868.html):

> Parameter names are lowercase by convention. The data type starts with a lowercase letter to indicate an object rather than a class. The description begins with a lowercase letter if it is a phrase (contains no verb), or an uppercase letter if it is a sentence. End the phrase with a period only if another phrase or sentence follows it.